support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -194,68 +200,3 @@ test.describe('Supporter Plus promoCodes', () => {
 		});
 	});
 });
-
-const thresholdTests = [
```

Comment:
I've removed these as they're really hard to get to work dynamically and I am not sure what confidence they are giving us?

Happy to find a way to re-add them if others think we should

Reply:
Agree they will fail every price move unfortunately, suppose they are set the same with every price change so perhaps donot need to be tested?

Reply:
Mostly it's a model we are no longer going to be using (we're using products `SupporterPlus` and `Contribution`).